Make tests run on Grsecurity system. This patch is downstream only.

--- a/Makefile
+++ b/Makefile
@@ -104,14 +104,17 @@
 $(O)/all_spec: $(DEPS) $(SOURCES) $(SPEC_SOURCES)
 	@mkdir -p $(O)
 	$(BUILD_PATH) ./bin/crystal build $(FLAGS) -o $@ spec/all_spec.cr
+	paxmark m $(O)/all_spec
 
 $(O)/std_spec: $(DEPS) $(SOURCES) $(SPEC_SOURCES)
 	@mkdir -p $(O)
 	$(BUILD_PATH) ./bin/crystal build $(FLAGS) -o $@ spec/std_spec.cr
+	paxmark m $(O)/std_spec
 
 $(O)/compiler_spec: $(DEPS) $(SOURCES) $(SPEC_SOURCES)
 	@mkdir -p $(O)
 	$(BUILD_PATH) ./bin/crystal build $(FLAGS) -o $@ spec/compiler_spec.cr
+	paxmark m $(O)/compiler_spec
 
 $(O)/crystal: $(DEPS) $(SOURCES)
 	@mkdir -p $(O)
